customer-node/service/ticketsService.js

```diff
@@ -0,0 +1,19 @@
+const {
+  findTicketsByCustomer, createTicket, findTicketById, deleteTicket,
+} = require('../data_access/ticketda');
+
+const getTicketsByCustomerId = (customerId) => findTicketsByCustomer(customerId);
+
+const addTicket = (ticket) => createTicket(ticket);
```

Comment:
Make sure to decrease seatsAvailable in flights here

Reply:
done